File: specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2023-06-01/examples-java/WorkbookDelete.java

/**
 * Samples for Workbooks Delete.
 */
public final class Main {
    /*
     * x-ms-original-file:
     * specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2023-06-01/examples/WorkbookDelete.
     * json
     */
    /**
     * Sample code: WorkbookDelete.
     * 
     * @param manager Entry point to ApplicationInsightsManager.
     */
    public static void
        workbookDelete(com.azure.resourcemanager.applicationinsights.ApplicationInsightsManager manager) {
        manager.workbooks().deleteByResourceGroupWithResponse("my-resource-group",
            "deadb33f-5e0d-4064-8ebb-1a4ed0313eb2", com.azure.core.util.Context.NONE);
    }
}

File: specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2023-06-01/examples-js/WorkbookDelete.js
const { ApplicationInsightsManagementClient } = require("@azure/arm-appinsights");
const { DefaultAzureCredential } = require("@azure/identity");
require("dotenv/config");

/**
 * This sample demonstrates how to Delete a workbook.
 *
 * @summary Delete a workbook.
 * x-ms-original-file: specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2023-06-01/examples/WorkbookDelete.json
 */
async function workbookDelete() {
  const subscriptionId =
    process.env["APPLICATIONINSIGHTS_SUBSCRIPTION_ID"] || "6b643656-33eb-422f-aee8-3ac145d124af";
  const resourceGroupName =
    process.env["APPLICATIONINSIGHTS_RESOURCE_GROUP"] || "my-resource-group";
  const resourceName = "deadb33f-5e0d-4064-8ebb-1a4ed0313eb2";
  const credential = new DefaultAzureCredential();
  const client = new ApplicationInsightsManagementClient(credential, subscriptionId);
  const result = await client.workbooks.delete(resourceGroupName, resourceName);
  console.log(result);
}
